system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -814,6 +816,93 @@ impl pallet_xcm_bridge_hub_router::Config<ToKusamaXcmRouterInstance> for Runtime
 	type FeeAsset = xcm_config::bridging::XcmBridgeHubRouterFeeAssetId;
 }
 
+pub type PoolAssetsInstance = pallet_assets::Instance3;
+impl pallet_assets::Config<PoolAssetsInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Balance = Balance;
+	type RemoveItemsLimit = ConstU32<1000>;
+	type AssetId = u32;
+	type AssetIdParameter = u32;
+	type Currency = Balances;
+	type CreateOrigin = NeverEnsureOrigin<AccountId>;
+	type ForceOrigin = AssetsForceOrigin;
+	type AssetDeposit = ConstU128<0>;
+	// TODO any?
+	type AssetAccountDeposit = ConstU128<0>;
```

Comment:
I think it's fine to have `AssetAccountDeposit` be `0` for the pools.

Reply:
When someone transfers lp tokens, the receiving account must have a provider and consumer references left, since the pool's lp asset class is not sufficient. or you should touch it. 
Zero for `AssetAccountDeposit` is not ok probably, since `assets.touch` call is permissionless and will create an account with no deposit. 
Non zero `AssetAccountDeposit` will set this deposit as an additional const for the pool creation. But we also can avoid this if we move `assets.touch` calls from `create_pool` to separate `asset-conversion.touch` call or to `asset-conversion.add_liquidity` call as I proposed in the related issue.

Reply:
> Zero for AssetAccountDeposit is not ok probably, since assets.touch call is permissionless and will create an account with no deposit.

Ah yeah you are right, so it should at least be the same as the Trust Backed Assets instance.

> Non zero AssetAccountDeposit will set this deposit as an additional const for the pool creation. But we also can avoid this if we move assets.touch calls from create_pool to separate asset-conversion.touch call or to asset-conversion.add_liquidity call as I proposed in the related issue.

I don't remember exactly what this entails, but IIRC I took a look and was OK with this. But we'd need to have this in a new release of `polkadot-sdk`, right? Perhaps we can set this non-zero now and implement this other solution to update this later?